client/common/cmdline.c

```diff
@@ -2108,18 +2108,105 @@ static int parse_kbd_options(rdpSettings* settings, const COMMAND_LINE_ARGUMENT_
 				else if (!freerdp_settings_set_bool(settings, FreeRDP_UnicodeInput, bval > 0))
 					rc = COMMAND_LINE_ERROR_UNEXPECTED_VALUE;
 			}
+#if defined(WITH_FREERDP_DEPRECATED_COMMANDLINE)
 			else if (count == 1)
 			{
 				/* Legacy, allow /kbd:<value> for setting keyboard layout */
 				rc = parse_kbd_layout(settings, val);
 			}
+#endif
+			else
+				rc = COMMAND_LINE_ERROR_UNEXPECTED_VALUE;
+
+			if (rc != 0)
+				break;
+		}
+	}
+	return rc;
+}
+
+static int parse_app_option_program(rdpSettings* settings, const char* cmd)
+{
+	const size_t ids[] = { FreeRDP_RemoteApplicationMode, FreeRDP_RemoteAppLanguageBarSupported,
+		                   FreeRDP_Workarea, FreeRDP_DisableWallpaper,
```

Comment:
Weird indentation, but if clang-format did it, well, then ignore this comment.

Reply:
yes, sometimes it is a bit strange, but better to have it this way than manual formatting all over the place.